docs/en/integrations/data-visualization/tableau/tableau-connection-tips.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_label: Connection Tips
+sidebar_position: 3
+slug: /en/integrations/tableau/connection-tips
+keywords: [clickhouse, tableau, online, mysql, connect, integrate, ui]
+description: Tableau connection tips when using ClickHouse official connector.
+---
+
+# Connection tips
+## Initial SQL tab
+If the *Set Session ID* checkbox is activated on the Advanced tab (by default), feel free to set session level [settings](https://clickhouse.com/docs/en/operations/settings/settings/) using
+```
+SET my_setting=value;
```

Comment:
What is the benefit of having it as a separate page? It might belong to `/en/integrations/tableau` to provide all the connection details on a single page.

Reply:
Mainly because it is relevant for both Desktop and Server usage (therefore I didn't want to put it on the Desktop page alone), and because it is less messy. 
If you strongly disagree with that, I don't mind adding this to the Tableau Desktop page.

Reply:
My concern is these tips will be harder to discover. We can add them to both pages or add links to the respective page